### PR TITLE
Updated KAZR Z_min with more accurate values; added the ENA KAZR props

### PR DESCRIPTION
**Pull request by isilber**

### CODE REVIEW

**Comment by rcjackson** [comment]
In the tests that I run for each pull request I include a style checker to ensure that contributed code is readable using the PEP8 standard. Therefore, if there are issues with style, the checker will fail with given warnings.

The style check failed on three lines of instruments.py:

./emc2/core/instruments.py:87:34: W291 trailing whitespace
./emc2/core/instruments.py:100:35: W291 trailing whitespace
./emc2/core/instruments.py:153:116: E501 line too long (127 > 115 characters)

What you'll need to do is remove the two trailing whitespaces and change line 153 to be less than 115 characters long. After that, this pull request should pass.

**Comment by isilber** [comment]
@rcjackson - I've incorporated these changes. Thanks!

File: emc2/core/model.py [hunk 1]
```diff
@@ -213,6 +232,17 @@ def __init__(self, file_path):
...
             self.ds = self.ds.squeeze(dim=('lat', 'lon'))
 
+        # crop specific model output time range (if requested)
+        if time_range is not None: 
+            try:
+                if np.issubdtype(time_range.dtype, np.datetime64):
+                    super()._crop_time_range(time_range)
+                else:
+                    raise RuntimeError("input time range is not in the required datetime64 data type")
+            except:
```
Comment (rcjackson) [inline emc2/core/model.py:242]: You'll have to find out here what type of error you want to catch. I highly recommend against using a blanket except statement here and PEP8 does not allow for a blanket except statement.

**Comment by rcjackson** [comment]
Some more issues with code style were brought up by the checker:

./emc2/core/instruments.py:153:70: W291 trailing whitespace
./emc2/core/model.py:111:41: E251 unexpected spaces around keyword / parameter equals
./emc2/core/model.py:186:45: E251 unexpected spaces around keyword / parameter equals
./emc2/core/model.py:186:47: E251 unexpected spaces around keyword / parameter equals
./emc2/core/model.py:236:35: W291 trailing whitespace
./emc2/core/model.py:242:13: E722 do not use bare 'except'
./emc2/core/model.py:244:1: W293 blank line contains whitespace
./emc2/core/model.py:246:9: E303 too many blank lines (2)

File: emc2/core/model.py [hunk 2]
```diff
@@ -164,7 +183,7 @@ def load_subcolumns_from_netcdf(self, file_name):
 
 
 class ModelE(Model):
-    def __init__(self, file_path):
+    def __init__(self, file_path, time_range = None):
```
Comment (rcjackson) [inline emc2/core/model.py:186]: For keyword arguments, there should be no space between the variables and the equal signs (see also line 111).

**Pull request merged**